frontend/src/scenes/actions/ActionEdit.js

```diff
@@ -165,59 +141,49 @@ export class ActionEdit extends Component {
                                 actionId={action.id}
                                 simmer={simmer}
                                 onDelete={() => {
-                                    action.steps = action.steps.filter(
-                                        s => s.id != step.id
-                                    )
+                                    action.steps = action.steps.filter(s => s.id != step.id)
                                     this.setState({ action: action })
                                 }}
                                 onChange={newStep => {
                                     action.steps = action.steps.map(s =>
-                                        (step.id && s.id == step.id) ||
-                                        (step.isNew && s.isNew === step.isNew)
+                                        (step.id && s.id == step.id) || (step.isNew && s.isNew === step.isNew)
                                             ? {
                                                   id: step.id,
                                                   isNew: step.isNew,
                                                   ...newStep,
                                               }
                                             : s
                                     )
-                                    this.setState({ action: action })
+                                    this.setState({ action: action, edited: true })
                                 }}
                             />
                         </>
                     ))}
 
                     <br />
 
-                    {this.state.saved && !isEditor && (
-                        <p className="text-success">Action saved.</p>
-                    )}
+                    {this.state.saved && !isEditor && <p className="text-success">Action saved.</p>}
```

Comment:
We've moved towards having Toasts to indicate success. Could we do `toast('Action saved')` and remove this line while we're at it?